### PR TITLE
Use prefixed commands rather than flags #63

### PR DESCRIPTION
**Pull request by andrey-starodubtsev**
- added prefixed commands with prefix g:simplenote_prefix
- fixed misprint in doc/simplenote.txt

### CODE REVIEW

**Comment by m1foley** [comment]
Thanks for doing this! If @mrtazz approves I'd like to have these prefixed commands be the default:

- Update the command references in `README` and `doc`
- Hardcode the command names instead of using `g:simplenote_prefix`

**Comment by atomicules** [comment]
I'll try to take a look over this weekend.

File: doc/simplenote.txt [hunk 1]
```diff
@@ -98,7 +98,7 @@ If set, the split windows will be opened vertical instead of horizontal. >
 
                                                   *'simplenote_single_window'*
 Default: 0
-If set, simplenote.vim will try to emulate the behaviout of the Simplenote
+If set, simplenote.vim will try to emulate the behaviour of the Simplenote
```
Comment (atomicules) [inline doc/simplenote.txt]: I make so many silly typos like this. Thanks for spotting.

**Comment by atomicules** [comment]
Actually made good progress soon. If I don't finish it up soon I'll push what I have to this pull request so you can at least see what I'm in the middle of doing.

**Comment by atomicules** [comment]
As promised, just pushed the work-in-progress stuff. Snowed under with work at the moment so it's taking me longer than expected to finish this off.

**Comment by atomicules** [comment]
What the hell, that branch doesn't seem to work at all now. I need some un-tired time to do this.

**Comment by atomicules** [comment]
Ok, I've started sorting out the mess I've made. I should hopefully get something pushed up later today.

Wooo... opensource can be embarrassing.

**Pull request merged**